hack/patches/scancode_ignore_binary.patch

```diff
@@ -0,0 +1,16 @@
+# Don't scan binary & media files
+# Ugly, not upstreamable hack
+
+diff -up scancode-toolkit-2.0.1/src/scancode/cli.py.ignore-binary scancode-toolkit-2.0.1/src/scancode/cli.py
+--- scancode-toolkit-2.0.1/src/scancode/cli.py.ignore-binary	2017-07-03 18:02:17.000000000 +0200
++++ scancode-toolkit-2.0.1/src/scancode/cli.py	2017-07-27 13:26:07.076096747 +0200
+@@ -621,6 +621,9 @@ def _scanit(paths, scanners, scans_cache
+     infos['path'] = rel_path
+     infos.update(scan_infos(abs_path, diag=diag))
+ 
++    if any((infos['is_binary'], infos['is_media'])):
```

Comment:
@jpopelka very nice! .... I am relating this to the ticket you entered in ScanCode https://github.com/nexB/scancode-toolkit/issues/689 to eventually have a built in pre-scan plugin that would have the same effect